src/TimeTrack.cpp

```diff
@@ -254,7 +254,7 @@ void TimeTrack::Draw(wxDC & dc, const wxRect & r, double h, double pps)
                             //
                             // LL:  It's because the ruler only Invalidate()s when the new value is different
                             //      than the current value.
-   mRuler->SetFlip(GetHeight() > 75 ? true : true); // MB: so why don't we just call Invalidate()? :)
+   mRuler->SetFlip(true);   // MB: so why don't we just call Invalidate()? :)
```

Comment:
The uninit'd variables.  Is the the one in NumericTextControl?  Be careful with that one.  I tried fixing it a while back:

[my attempt](https://code.google.com/p/audacity/source/detail?r=13138&path=/audacity-src/trunk/src/widgets/TimeTextCtrl.cpp)

It didn't work.  :-)

[fail](http://sourceforge.net/p/audacity/mailman/message/32414338/)

It'll be good if you got it squashed this time.